src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -60,10 +61,21 @@
             <tag name="serializer.normalizer" />
         </service>
 
-        <!-- Resource path naming strategies -->
+        <!-- Resources Operations path resolver -->
+
+        <service id="api_platform.operation_path_resolver" alias="api_platform.operation_path_resolver.router" public="false" />
+
+        <service id="api_platform.operation_path_resolver.router" class="ApiPlatform\Core\Bridge\Symfony\Routing\RouterOperationPathResolver" public="false">
+            <argument type="service" id="api_platform.router" />
+            <argument type="service" id="api_platform.operation_path_resolver.custom" />
+        </service>
+
+        <service id="api_platform.operation_path_resolver.custom" class="ApiPlatform\Core\PathResolver\CustomOperationPathResolver" public="false">
```

Comment:
Shouldn't `api_platform.operation_path_resolver.custom` decorate `api_platform.operation_path_resolver.default`?

(Why do we need the "default" part of the name, in the first place? This was already discussed before in #547)